cmd/cni-metrics-helper/metrics/metrics.go

```diff
@@ -397,13 +401,13 @@ func metricsListGrabAggregateConvert(t metricsTarget) (map[string]*dto.MetricFam
 	resetMetrics(interestingMetrics)
 
 	targetList := t.getTargetList()
-	glog.Info("targetList: ", targetList)
-	glog.Info("len(targetList)", len(targetList))
+	log.Infof("targetList: %v ", targetList)
+	log.Infof("len(targetList) %d", len(targetList))
 	for _, target := range targetList {
-		glog.Infof("Grab/Aggregate metrics from %v", target)
+		log.Infof("Grab/Aggregate metrics from %v", target)
 		rawOutput, err := t.grabMetricsFromTarget(target)
 		if err != nil {
-			glog.Errorf("Failed to getMetricsFromTarget: %v", err)
+			log.Errorf("Failed to getMetricsFromTarget: %v", err)
```

Comment:
Should we make this a `log.Warnf()` instead? Ref to #592. (I think we can remove [this line](https://github.com/aws/amazon-vpc-cni-k8s/blob/8340e95126da40e8489074c68052869c4cd098d2/cmd/cni-metrics-helper/metrics/cni_metrics.go#L158) as well.)

Reply:
Addressed.